reference/eu-domain.mdx

```diff
@@ -1,24 +1,24 @@
 ---
-title: 'EU domain'
-description: 'This page explains how to work with Axiom if your organization uses the EU domain.'
+title: 'EU region'
```

Comment:
Would expect the title of this page to be Regions; EU just happens to be the less common case, but might not always be that way